src/v2/Apps/Fair/Routes/FairArtworks.tsx

```diff
@@ -42,9 +45,16 @@ const FairArtworksFilter: React.FC<FairArtworksFilterProps> = props => {
 
   const { counts } = filtered_artworks
 
+  // TODO: You shouldn't have to pass `relayEnvironment` and `user` through below.
+  // For some reason, they are undefined when `useSystemContext()` is referenced
+  // in <ArtistsFilter />. So, pass as props for now.
   const Filters = () => (
     <Box pr={2}>
-      <ArtistsFilter />
+      <ArtistsFilter
+        fairID={fair.internalID}
+        relayEnvironment={relayEnvironment}
```

Comment:
Unfort. needed to prop drill these, the `useSystemContext` hook was returning some `undefined`s when used in the filter (some bug/interaction with the `<ArtworkFilterContextProvider>`?).

Reply:
Yeah this is super mysterous, no clue why this wouldn't work. I've seen issues like we observed with circular references, but otherwise its unexpected. I can poke at it once its merged.